src/dra/843B0.c

```diff
@@ -3126,15 +3124,15 @@ void func_8012C600(void) {
             x = g_Entities[i + 21].posX.i.hi + 4;
         }
     }
-    D_8013AEBC = x - t0;
```

Comment:
Are these consecutive on psp? I think this is in the bss section so on psx they're in a different order than they were originally declared

Reply:
D_8013AEBC[0] is D_psp_09236714
D_8013AEBC[1] is D_psp_09236718
D_8013AEBC[2] is D_psp_0923671C
D_8013AEBC[3] is D_psp_09236720

So yes, looks like they are consecutive and are in the same order on both systems.